nemo_curator/modules/__init__.py

```diff
@@ -36,6 +36,16 @@
 FuzzyDuplicates = gpu_only_import_from(
     "nemo_curator.modules.fuzzy_dedup", "FuzzyDuplicates"
 )
+SemDedup = gpu_only_import_from("nemo_curator.modules.semantic_dedup", "SemDedup")
+EmbeddingCreator = gpu_only_import_from(
+    "nemo_curator.modules.semantic_dedup", "EmbeddingCreator"
+)
+ClusteringModel = gpu_only_import_from(
+    "nemo_curator.modules.semantic_dedup", "ClusteringModel"
+)
+SemanticClusterLevelDedup = gpu_only_import_from(
+    "nemo_curator.modules.semantic_dedup", "SemanticClusterLevelDedup"
+)
 
 # Pytorch related imports must come after all imports that require cugraph,
 # because of context cleanup issues b/w pytorch and cugraph
```

Comment:
Nit: Move the comment above semdedup imports since these modules also import PyTorch and that's the reason it's placed after cugraph imports.

Reply:
Moved: https://github.com/NVIDIA/NeMo-Curator/pull/130/commits/12373a7e4f7377b36dab6cc81dae94907a18da95